build.sbt

```diff
@@ -65,13 +65,28 @@ ThisBuild / developers := List(
   tlGitHubDev("rossabaker", "Ross A. Baker")
 )
 
+ThisBuild / tlJdkRelease := Some(11)
 ThisBuild / githubWorkflowJavaVersions := Seq("11", "17").map(JavaSpec.temurin(_))
 ThisBuild / tlCiReleaseBranches := Seq("series/0.7")
 ThisBuild / tlSitePublishBranch := Some("series/0.7")
 
 lazy val docsSettings =
   Seq(
-    fork := true,
+    tlSiteApiModule := Some((core / projectID).value),
+    tlSiteApiPackage := Some("org.http4s.jdkhttpclient"),
+    tlSiteHeliumConfig ~= {
+      import laika.rewrite._
+      _.site.versions(
+        Versions(
+          currentVersion = Version("0.7.x", "0.7"),
+          olderVersions = Seq(
+            Version("0.5.x", "0.5.0"),
+            Version("0.4.x", "0.4.0")
+          ),
```

Comment:
Is it weird to exclude `0.6.x`? It never got a stable release in the end 🙃

Reply:
Let's just add it, I don't think there is a downside.

Reply:
👍 I guess we can point it at the latest milestone.

Reply:
```suggestion
          olderVersions = Seq(
            Version("0.6.x", "0.6.0-M7"),
            Version("0.5.x", "0.5.0"),
            Version("0.4.x", "0.4.0")
          ),
```

Reply:
FTR, it actually doesn't matter that we make this change here. All the versioning will be controlled from the main branch anyway.